draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -685,6 +697,13 @@ are dispatched during the ICE candidate gathering and processing described in
 ensure a single origin does not prevent other origins from registering,
 unregistering, or resolving mDNS names.
 
+External mDNS queries
+----------------------------------------
+
+The browser is expected to answer mDNS query messages, which might have the
```

Comment:
How would the client do this though - is there a ref we can point at?

Generally, I think this attack needs a bit more discussion to make it clear how this can happen and what specifically we recommend implementations to do.

Reply:
6762 S5.5 has a very brief account on this:

> responders SHOULD check that the source address in the query packet matches the local subnet for that link (or, in the case of IPv6, the source address has an on-link prefix)

This may serve as a ref but agreed we need to expand on this attack. Given also the ongoing discussion on whether we can bootstrap trust from any unicast transactions, it seems we need to explicitly disallow unicast queries in this spec.

Reply:
Removed the section, filed https://github.com/rtcweb-wg/mdns-ice-candidates/issues/117 to continue the discussion.